### DISCUSSION TITLE
Payment Method id not working in my site

### DISCUSSION THREAD

**Discussion by atiaparvin763** [woocommerce/woocommerce#51699] [Checkout Flow]
Payment Method id not working in my site. What to do please suggest.

**Comment by atiaparvin763** [comment]
https://www.istitch.xyz/?post_type=product

**Comment by mikejolley** [comment]
Hi,

If you're having issues with a specific payment plugin I would suggest going through the resources that payment plugin offers or using its support. For general questions with WooCommerce and its core payment options you can use this forum:

https://wordpress.org/support/plugin/woocommerce/

Without further details there is not much support we can offer here. 

If you're moving from shortcode checkout to blocks checkout and finding the payment method is not compatible, currently it’s up to extension developers to add support for the new checkout flow offered via the Cart and Checkout Blocks. We have some [documentation for Third-party developers](https://href.li/?https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce-blocks/docs#third-party-developers) and some [Developer Resources](https://href.li/?https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce-blocks/docs#developer-resources) available to help with their integration efforts.

Your best course of action now if you are using a payment method that isn’t working with the blocks is to notify the developer of the extension and ask if they can add support.

Thanks
